config_examples/dblp_mapping.json

```diff
@@ -0,0 +1,47 @@
+{
+  "nodes": [
+    {
+      "nodeMappingId": "nm_author",
+      "nodeLabel": "Author",
+      "xpath": "/dblp/article/author",
+      "innerTextMappedTo": "name",
+      "mappedAttributes": {},
+      "mappedChildElements": {}
+    },
+    {
+      "nodeMappingId": "nm_publication",
+      "nodeLabel": "Publication",
+      "xpath": "/dblp/article",
+      "innerTextMappedTo": null,
+      "mappedAttributes": {
+        "publtype": "type"
+      },
+      "mappedChildElements": {
+        "title": "title",
+        "year": "year"
+      }
+    }
+  ],
+  "edges": [
+    {
+      "edgeLabel": "CollaboratedWith",
+      "directed": false,
+      "firstNodeMappingId": "nm_author",
+      "secondNodeMappingId": "nm_author",
+      "mappedAttributes": {
+        "mdate": "date"
+      },
```

Comment:
Nie wiadomo z którego tagu brać te atrybuty. Rozumiem, że domyślnie ze wspólnego rodzica, ale można by też brać z dziecka. Przypadku, gdy chcę zrobić krawędź między publikacjami ze wspólnym autorem, gdzie atrybutem ma być nazwisko autora to już nie obsłuży.